zone/trading.cpp

```
@@ -2450,6 +2427,61 @@ void Client::SellToBuyer(const EQApplicationPacket *app)
+
+	if (!GetInv().HasSpaceForItem(item, static_cast<int16>(quantity))) {
+		LogTrading(
+			"Buyer [{}] has insufficient inventory space for item [{}] quantity [{}]",
+			GetCleanName(),
+			item_id,
+			quantity
+		);
+		return false;
+	}
+
+	if (item->Stackable) {
```

Cppcheck (2.21.0)

[style] The function 'SetBotPulling' is never used.

(unusedFunction)